Typescript error on navigationView

When I am running SampleApp using yarn I am receiving below typescript error for `node_modules/react-native-navigation-sdk/components/navigation/navigationView/index.tsx:364:43`

NavSDK version: 0.3.0-beta

Error: 
```
yarn run v1.22.22
$ tsc && react-native run-ios
node_modules/react-native-navigation-sdk/components/navigation/navigationView/index.tsx:364:43 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ onRouteChanged: () => void; onRemainingTimeOrDistanceChanged: () => void; onTrafficUpdated: () => void; onArrival: (args: ArrivalEvent) => void; onNavigationReady: () => void; ... 12 more ...; logDebugInfo: (message: string) => void; }'.
  No index signature with a parameter of type 'string' was found on type '{ onRouteChanged: () => void; onRemainingTimeOrDistanceChanged: () => void; onTrafficUpdated: () => void; onArrival: (args: ArrivalEvent) => void; onNavigationReady: () => void; ... 12 more ...; logDebugInfo: (message: string) => void; }'.

364       eventEmitter.addListener(eventName, this.nativeEventsToCallbackMap[eventName]);
                                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/react-native-navigation-sdk/components/navigation/navigationView/index.tsx:364

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command
```

--- a/components/navigation/navigationView/index.tsx
+++ b/components/navigation/navigationView/index.tsx
@@ -41,7 +41,7 @@ import {ArrivalEvent, NavigationViewProps} from './types';
 export default class NavigationView extends React.Component<NavigationViewProps> {
   private viewId: number = -1;
   private mapViewRef?: any;
-  private nativeEventsToCallbackMap;
+  private nativeEventsToCallbackMap: { [key: string] : (event: any) => void };
 
   constructor(_props: NavigationViewProps) {
     super(_props);
@@ -361,7 +361,10 @@ export default class NavigationView extends React.Component<NavigationViewProps>
     );
 
     for (const eventName of Object.keys(this.nativeEventsToCallbackMap)) {
-      eventEmitter.addListener(eventName, this.nativeEventsToCallbackMap[eventName]);
+      const listener = this.nativeEventsToCallbackMap[eventName];
+      if (listener != undefined) {
+        eventEmitter.addListener(eventName, listener);
+      }
     }
   };
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-native-navigation-sdk",
-  "version": "0.3.0-beta",
+  "version": "0.3.1-beta",
   "author": "Google",
   "description": "A react-native library for Google's Navigation SDK",
   "private": false,

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -18,8 +18,8 @@
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitOverride": true,
-    "allowUnreachableCode": false,
-    "noImplicitAny": false,
+    "allowUnreachableCode": true,
+    "noImplicitAny": true,
     "moduleSuffixes": [".android", ".ios", ""]
   },
   "include": [

